common/models/api.js

```diff
@@ -6,7 +8,7 @@ module.exports = function api(Api) {
       isStatic: true,
       accepts: [],
       returns: {arg: 'response', type: 'object', root: true},
-      description: 'Shutdown this server.',
+      description: g.t('Shutdown this server.'),
```

Comment:
That doesn't make sense to me. This is the documentation string that will show up in the explorer.

- did you test this works?
- since almost all the API is defined by JSON files, and they are not translated, won't this be bizarre even if it does work, to have an almost entirely English explorer, with only the remote methods have localized string descriptions?